lib/modules/manager/pep621/schema.ts

```diff
@@ -55,3 +57,18 @@ export const PyProjectSchema = z.object({
     })
     .optional(),
 });
+
+export const PdmLockfileSchema = Toml.pipe(
+  z.object({
+    package: LooseArray(
+      z
+        .object({
+          name: z.string(),
+          version: z.string(),
+        })
+        .transform(({ name, version }): [string, string] => [name, version]),
```

Comment:
why this double transformation to array and back to object? 🤔

Reply:
@Churro Can we simplify this schema? it looks unnecessary complex.

Reply:
It seems I'm still not best friends with `zod`. Thank you for helping out 😊